projects/media-viewer/src/lib/annotations/annotation-set/annotation/comment/comment.component.ts

```diff
@@ -35,6 +36,34 @@ export class CommentComponent {
     this.fullComment = comment.content;
   }
 
+  @Input()
+  set selected(selected: boolean) {
```

Comment:
we'll this is confusing

Reply:
it's so that we can take into account the logic for draft comments, so that they remain selected until they press save